gui/src/app/pages/HomePage/HomePage.tsx

```diff
@@ -143,6 +136,60 @@ const HomePageChild: FunctionComponent<Props> = ({ width, height }) => {
   );
 };
 
+type RightViewProps = {
+  width: number;
+  height: number;
+  compiledMainJsUrl: string;
+};
+
+const tabs = [
+  {
+    id: "sampling",
+    label: "Sampling",
+    title: "Run sampling and view draws",
+    closeable: false,
+  },
+  {
+    id: "analysis.py",
+    label: "Analysis (Py)",
+    title: "Python analysis",
+    closeable: false,
+  },
+  {
+    id: "analysis.r",
+    label: "Analysis (R)",
+    title: "R analysis",
+    closeable: false,
+  }
+];
+
+const RightView: FunctionComponent<RightViewProps> = ({
+  width,
+  height,
+  compiledMainJsUrl,
+}) => {
+  const [currentTabId, setCurrentTabId] = useState("sampling");
+  return (
+    <TabWidget
+      width={width}
+      height={height}
+      tabs={tabs}
+      currentTabId={currentTabId}
+      setCurrentTabId={setCurrentTabId}
+    >
+      <SamplingWindow
+        width={width}
+        height={height}
+        compiledMainJsUrl={compiledMainJsUrl}
+      />
+      <AnalysisPyWindow width={width} height={height} />
+      <div>
+        R Analysis not yet implemented
+      </div>
+    </TabWidget>
+  );
+};
```

Comment:
The tabs have since been reorganized in response to Brian's suggestion. So I'll resolve this particular conversation, although I do agree that we should start splitting components out of some of the larger files.